app/assets/javascripts/scripts.js

```diff
@@ -311,6 +311,22 @@ function iAgree(){
      $('table.display').dataTable({"sPaginationType": "full_numbers", "iDisplayLength" : 10, ordering: true});
  });
 
+ /*
+  * ============================================================================
+  * Collection_select Code
+  * ============================================================================
+  */
+
  $(document).ready(function() {
   $('select').material_select();

```

Comment:
can you move this to the top of the file next to the rest of the other materialize functions